draft-ietf-6man-slaac-renum-08.xml

```diff
@@ -539,6 +664,9 @@ In particular:
 Routers MAY be explicitly or implicitly configured to send multiple RAs and split information between them. For example, a router could be configured to send information associated with different provisional domains <xref target="RFC7556"/> in different RAs, or to send multiple RAs, one per VRRPv3 <xref target="RFC9568"/> group.
 </t>
 <t>
+If prefix-specific link-local addresses [draft-ietf-6man-slaac-renum] are enabled for the interface, the router SHOULD include one PIO into each RA, as described in [draft-ietf-6man-slaac-renum].
+</t>
```

Comment:
What about the rest of the information? should it be replicated into each RA? (e.g., RDNSS)

Reply:
As it doesn't matter I do not think we shall be (over) specifying.

Reply:
The issue here is that by doing gulla, you turn flash-renum into a, say, multi-homing (multi-ipv6) problem. IN the context of https://www.ietf.org/archive/id/draft-gont-v6ops-multi-ipv6-01.txt , the RA should include all the information that is usable with that router.  (Otherwise, how would a hsot infer that it is okay to use, say, RDNSS addresses with this new router?)